<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>乐理精灵</string>
  </property>
  <property name="windowIcon">
   <iconset resource="katmus.qrc">
    <normaloff>:/images/iconpent.png</normaloff>:/images/iconpent.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="QWidget" name="widget_toolbox" native="true">
      <layout class="QGridLayout" name="gridLayout_toolbox">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QWidget" name="widget_keybd" native="true">
      <layout class="QGridLayout" name="gridLayout_keybd">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QWidget" name="widget_chord" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="btn_0">
         <property name="text">
          <string>大调主三</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_1">
         <property name="text">
          <string>小调下属三</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btn_2">
         <property name="text">
          <string>小调属三</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btn_3">
         <property name="text">
          <string>大调下属三</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="btn_4">
         <property name="text">
          <string>大调属三</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="btn_5">
         <property name="text">
          <string>小调主三</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QPushButton" name="btn_6">
         <property name="text">
          <string>小调下属七</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QPushButton" name="btn_7">
         <property name="text">
          <string>小调属七</string>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QPushButton" name="btn_8">
         <property name="text">
          <string>大调下属增七</string>
         </property>
        </widget>
       </item>
       <item row="0" column="9">
        <widget class="QPushButton" name="btn_9">
         <property name="text">
          <string>大调属七</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionOpenMidi"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>控制</string>
    </property>
    <addaction name="actionMIDI_controller"/>
    <addaction name="separator"/>
    <addaction name="undoAction"/>
    <addaction name="redoAction"/>
    <addaction name="separator"/>
    <addaction name="chordAnalyse"/>
    <addaction name="actionPlayMidi"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionOpenMidi"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="chordAnalyse"/>
   <addaction name="actionPlayMidi"/>
   <addaction name="actionMIDI_controller"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionMIDI_controller">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="katmus.qrc">
     <normaloff>:/images/piano.png</normaloff>:/images/piano.png</iconset>
   </property>
   <property name="text">
    <string>MIDI控制</string>
   </property>
   <property name="toolTip">
    <string>MIDI控制</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="katmus.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存乐谱</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="icon">
    <iconset resource="katmus.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="redoAction">
   <property name="icon">
    <iconset resource="katmus.qrc">
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>重复</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="chordAnalyse">
   <property name="icon">
    <iconset resource="katmus.qrc">
     <normaloff>:/images/wave.png</normaloff>:/images/wave.png</iconset>
   </property>
   <property name="text">
    <string>和弦分析</string>
   </property>
   <property name="toolTip">
    <string>和弦分析</string>
   </property>
  </action>
  <action name="actionPlayMidi">
   <property name="icon">
    <iconset resource="katmus.qrc">
     <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
   </property>
   <property name="text">
    <string>播放MIDI</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="katmus.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>加载乐谱</string>
   </property>
  </action>
  <action name="actionOpenMidi">
   <property name="icon">
    <iconset resource="katmus.qrc">
     <normaloff>:/images/iconproject.png</normaloff>:/images/iconproject.png</iconset>
   </property>
   <property name="text">
    <string>加载MIDI</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="katmus.qrc"/>
 </resources>
 <connections/>
</ui>
